tools/spelling/spelling_dictionary.txt

```diff
@@ -1432,4 +1434,4 @@ NSS
 SSLKEYLOGFILE
 DLB
 PCIE
-EDNS
+EDNS
```

Comment:
Do we need this? Whedre is this sequence in the PR?

Reply:
I an not sure I added that. 
might be from the merge. 
remove it.
I did see reference to EDNS in dns_filter/dns_parser.cc